### ISSUE TITLE
questions about example data

### ISSUE THREAD

**Issue by xinyiLLL** [Tianshu996/METSC#3]
I wonder the meaning of _example/data.mat,_ have you done **proprocessing** between input the data into METSC?
I got HCP dataset from authorized website, whose data format is **.nii.gz** , what's the difference between these two data?
Thanks!

**Comment by Tianshu996** [comment]
Thank you for your interest. 

The .mat file contains data extracted from .nii files. 

Do you mean 'preprocessing'? Whether preprocessing is required depends on your specific needs.

**Comment by xinyiLLL** [comment]
Yes! I mean preprocessing. I want to reproduce METSC code using the HCP dataset and would like to know what preprocessing is needed?

**Comment by xinyiLLL** [comment]
Besides, as for the details of model training, I wonder whether the input of the METSC model is a multi-layer two-dimensional patch along the Z-axis (e.g. transfer original HCP dataset _145 * 174 * 145_ to _25230 * 3 * 3 * 145_) or a three-dimensional volume (_3,658,350 *3 * 3 * 3_)

**Comment by Tianshu996** [comment]
In this demo, we use 2D patches as input. For example, in the provided data tensor with shape (3 × 3 × 25230 × 63), the dimension of size 63 combines both the diffusion features and the label information.

You can directly fine-tune or distill the provided pre-trained weights on your own dataset, even with different preprocessing methods.
